Разметка всех страниц

ElenaS63 opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2049601/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2049601-sedona-34/2/

ElenaS63 committed:
Добавляет шапку

ElenaS63 committed:
Исправляет шапку и выравнивание разметки

Убирает второй тег nav (навигация остается в одном теге).
Исправляет выравнивание разметки.

ElenaS63 committed:
Добавляет класс "кнопка" элементу навигации

ElenaS63 committed:
Добавляет заголовок страницы и список преимуществ

ElenaS63 committed:
Добавляет подвал главной страницы

ElenaS63 committed:
Добавляет список в мейн, лейбл для поля подписки

ElenaS63 committed:
Добавляет шапку и подвал

ElenaS63 committed:
Добавляет класс логотипу в шапке

ElenaS63 committed:
Добавляет в каталог блок с заголовком и хлебными крошками

ElenaS63 committed:
Добавляет логотипу в шапке ссылку

ElenaS63 committed:
Добавляет блок с фильтрами

ElenaS63 committed:
Добавляет сортировку

ElenaS63 committed:
Добавляет 1 карточку гостиницы

ElenaS63 committed:
добавляет карточки с отелями

ElenaS63 committed:
добавляет список пагинации

ElenaS63 committed:
Исправляет последовательность

Ustincheg commented on index.html:
```diff
@@ -1,12 +1,236 @@
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+      <ul class="navigation-list navigation-user">
```
вот этот список предлагаю вынести из тега nav

ElenaS63 replied:
исправлено

Ustincheg commented on index.html:
```diff
@@ -1,12 +1,236 @@
+        </ul>
+      </li>
+    </ul>
+    <h2>Седона — небольшой городок в Аризоне,
```
по смыслу это не заголовок, просто текст в секции преимуществ

ElenaS63 replied:
исправлено

Ustincheg commented on index.html:
```diff
@@ -1,12 +1,236 @@
+      заслуживающий большего!
+    </h2>
+    <h3>
+      Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!
```
просто текст

ElenaS63 replied:
исправлено

Ustincheg commented on index.html:
```diff
@@ -1,12 +1,236 @@
+          <p class="advantages-description">
+            Седона — не аттракцион для туристов, там течёт своя жизнь
+          </p>
+          <img class="photo-1" src="#" alt="Фото города">
```
эту картинку потом фоном подключим, пока можно удалить из разметки

ElenaS63 replied:
исправлено

Ustincheg commented on index.html:
```diff
@@ -1,12 +1,236 @@
+    </section>
+    <section class="hotel-search">
+      <h2 class="hotel-search-title">
+        Заинтересовались?
```
это просто текст, следует использовать тег p

ElenaS63 replied:
исправлено

Ustincheg commented on index.html:
```diff
@@ -1,12 +1,236 @@
+        </li>
+      </ol>
+    </section>
+    <section class="hotel-search">
```
скрытый заголовок должен соответствовать смыслу секции. "Поиск гостиницы в Седоне"

ElenaS63 replied:
исправлено

ElenaS63 committed:
Исправляет разметку по замечаниям

1. Пользовательское меню вынесено из навигации;
2,3,4. Список преимуществ: текст обернут в тег р и добавлен в секцию преимуществ (добавлены классы), изображения убраны из разметки.
5,6. секция Поиск гостиницы в Седоне: добавлен скрытый заголовок и текст обернут в тег р.
